GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
-assembly-versioning-scheme: MajorMinorPatch 
-#next-version: 1.0.0
```

Comment:
@AArnott the version isn't stored anywhere, unless you are specifically trying to bump to a different, higher version.  The version number is asserted based on the current state of the repository, based on a number of possible things, i.e. current branch, tags, number of commits, commit messages, etc.

If I am not mistaken, I think @GeertvanHorrik was just locally running GitVersion, prior to doing a release.

The idea is that as part of your build, either through the task, or running the exe directly using some form of script, i.e. cake, psake, fake, rake, etc, you would run GitVersion, it would stamp the necessary files as required, and off you go.